dashboard/config/honeybadger.yml

```diff
@@ -0,0 +1,7 @@
+---
+api_key: "#{CDO.dashboard_honeybadger_api_key}"
+
+exceptions.ignore:
+  <%  %w(Sinatra StorageApps DynamoTable PropertyBag).each do |class_name| %>
+  - <%= class_name + "::NotFound" %>
+  <% end %>
```

Comment:
I don't know.  This is the existing behavior migrated from honeybadger.rb by the config migration script.